@here/harp-map-theme/resources/berlin_tilezen_base.json

```diff
@@ -86,19 +91,125 @@
     "styles": {
         "tilezen": [
             {
-                "description": "highway-label",
-                "when": "$layer == 'roads' && ((kind == 'highway') && ((kind_detail == 'motorway')))",
-                "technique": "text",
+                "description": "highway-roadshield-2",
+                "when": "$layer == 'roads' && (kind == 'highway') && (kind_detail == 'motorway') && has(ref) && (length(ref) <= 2)",
+                "technique": "line-marker",
                 "attr": {
-                    "color": "#000000",
-                    "backgroundColor": "#FFFFFF",
-                    "backgroundOpacity": 0.5,
-                    "size": 16,
-                    "priority": 25,
-                    "fadeNear": 0.8,
-                    "fadeFar": 0.9
+                    "style": "smallSign",
+                    "label": "ref",
+                    "size": 10,
+                    "imageTexture": "default-2",
+                    "iconScale": 1,
+                    "priority": 64,
+                    "minDistance": 200,
+                    "vAlignment": "Center",
+                    "hAlignment": "Center",
+                    "textIsOptional": false,
+                    "iconIsOptional": false,
+                    "textMayOverlap": false,
+                    "iconMayOverlap": false,
+                    "iconReserveSpace": true,
+                    "renderTextDuringMovements": true,
```

Comment:
> renderTextDuringMovements

Is this possible, that this one is responsible for severe flickering during mouse interactions ?

Reply:
I guess that rather, occlusion test:
 "iconMayOverlap": false,
causes the flicker, because changing it on POIs causes the same issue.